CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0-beta.1
```

Comment:
Is it customary the bump the major revision when we update the SDK constraint?

The pub solver will opt out people automatically from a 2.5.0 if they are not on 2.13.

And we're not doing an breaking changes I believe.

Reply:
Right.